### ISSUE TITLE
Step 2 firebase project name doesn't match the screenshot in Step 6 any more

### ISSUE THREAD

**Issue by MrCsabaToth** [firebase/assistant-codelab#2]
Our firebase project in Step 2 is "animal-guesser" (https://codelabs.developers.google.com/codelabs/assistant-codelab/index.html). In Step 6 during `firebase init` you have to select that same project, not the `chatbot`. Not a biggie but it may throw some complete beginners off.

**Comment by Nisar222** [comment]
Yes, I believe if you have existing firebase projects, you need to specify the project ID using the "--project parameter". 

example:
firebase deploy --project animal-guesser-16xx6

**Comment by MrCsabaToth** [comment]
@Nisar222 Correct. My intention is to synchronize the lab text with the lab screenshot so newcomers won't bump into name mismatch.
